### PR TITLE
Added exists api for password strength and profile.

### PR DESCRIPTION
**Pull request by ragupta-git**
Added exists api for password strength and profile.

### CODE REVIEW

File: ucsm_apis/admin/user.py [hunk 1]
```diff
@@ -520,6 +520,34 @@ def password_strength_check(handle, policy_owner = "local", descr=None,
     return mo
 
 
+def password_strength_exists(handle, **kwargs):
```
Comment (vvb) [inline ucsm_apis/admin/user.py]: `password_strength_exists` -> `password_strength_check_exists`

Comment (ragupta-git) [inline ucsm_apis/admin/user.py]: Done.

File: ucsm_apis/admin/user.py [hunk 2]
```diff
@@ -211,7 +221,7 @@ def user_role_add(handle, user_name, name, descr=None, **kwargs):
     Args:
         handle (UcsHandle)
         user_name (string): username
-        name (string): rolename
+        name (string): rolename, also allowed comma separated roles
```
Comment (ragupta-git) [inline ucsm_apis/admin/user.py]: rolename, also allowed comma separated roles -> rolename or a comma separated string of roles

File: ucsm_apis/admin/user.py [hunk 3]
```diff
@@ -263,14 +277,23 @@ def user_role_get(handle, user_name, name, caller="user_role_get"):
...
+
 def user_role_exists(handle, user_name, name, **kwargs):
     """
     check if role is already added to user
 
     Args:
         handle (UcsHandle)
         user_name (string): username
-        name (string): rolename
+        name (string): rolename, also allowed comma separated roles
```
Comment (ragupta-git) [inline ucsm_apis/admin/user.py]: same as above.

**Comment by ragupta-git** [comment]
Incorporated review comments

**Pull request merged**